src/main/java/uk/ac/ebi/ampt2d/registry/entities/Phenotype.java

```diff
@@ -47,28 +48,67 @@
         HEPATIC
     }
 
-    @ApiModelProperty(position = 1)
+    public enum Type {
+        DICHOTOMOUS,
+        MULTICHOTOMOUS,
+        CONTINUOUS
+    }
+
+    @ApiModelProperty(position = 1, required = true)
     @JsonProperty
     @NotNull
     @Size(min = 1, max = 255)
     @Id
     @Column(nullable = false, unique = true, updatable = false)
     private String id;
 
-    @ApiModelProperty(position = 3)
+    @ApiModelProperty(position = 2, required = true)
     @JsonProperty
     @NotNull
     @Enumerated(EnumType.STRING)
     @Column(nullable = false)
     private Group phenotypeGroup;
 
+    @ApiModelProperty(position = 3, required = true)
+    @NotNull
+    @NotBlank
+    @JsonProperty
+    @Column(nullable = false, columnDefinition = "TEXT")
+    private String description;
+
+    @ApiModelProperty(position = 4, required = true)
+    @JsonProperty
+    @NotNull
+    @Enumerated(EnumType.STRING)
+    @Column(nullable = false)
+    private Type type;
+
+    @ApiModelProperty(position = 5, required = true)
+    @JsonProperty
+    @NotNull
+    @Column(nullable = false)
+    private String allowedValues;
+
     @CreatedDate
     @Column(updatable = false)
+    @Convert(converter = ZonedDateAttributeConverter.class)
```

Comment:
Have you tried `@Temporal(TIMESTAMP)` annotation? I believe that will convert between `Date` and `TIMESTAMP` in database.

Reply:
we can use it for util. Date and Calendar i guess

Reply:
Or alternatively, use `java.sql.Timestamp` directly as the type for the creation date?

Reply:
as we have the registry shared between two timezones , we used this ZonedDateTime

Reply:
That actually makes a case for the `Timestamp` which is independent of timezones.

Reply:
i have used Temporal with date that solves the convertion